### PR TITLE
Update kenny.json

### PR DESCRIPTION
**Pull request by KennySB-dev**
kennysb-dev.github.io
due to stopped hosting on is-a.dev

### CODE REVIEW

**Comment by KennySB-dev** [comment]
+ i fixed my github name

File: domains/kenny.json [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
     },
     "record": {
-        "A": [
-            "217.174.245.249",
-            "51.161.54.161"
-            ],
+        "CNAME": "kennysb-dev.github.io",
         "MX": ["mail.is-a.dev"],
         "TXT": "v=spf1 mx a:mail.is-a.dev ~all"
     }
```
Comment (CuteDog5695) [inline domains/kenny.json:10]: sorry but you cant use CNAME with MX and TXT

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/kenny.json (hunk 1 above)
Comment (KennySB-dev) [reply domains/kenny.json:10]: wait so how can i use my own site hosted on github, and still use the mail hosted by is-a.dev?

Comment (CuteDog5695) [reply domains/kenny.json:10]: you can use A records with github pages, i use github pages with A record (tho you need to look up the A records for your github page)

Comment (KennySB-dev) [reply domains/kenny.json:10]: so instead of cname ill use a records

Comment (CuteDog5695) [reply domains/kenny.json:10]: mhm, look up the A records for kennysb-dev.github.io then replace CNAME with A records

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by KennySB-dev** [comment]
what do i do if its a merge conflict

**Pull request closed without merging**